sections/main-article.liquid

```diff
@@ -48,16 +48,48 @@
               {{ article.content }}
           </div>
 
-        {%- when 'social_sharing'-%}
+        {%- when 'share' -%}
           <div class="article-template__social-sharing page-width page-width--narrow" {{ block.shopify_attributes }}>
-            {% render 'social-sharing', share_title: article.title, share_permalink: article.url, share_image: article.image %}
+            <share-button class="share-button" {{ block.shopify_attributes }}>
+              <button class="share-button__button button button--tertiary hidden">
+                {% render 'icon-share' %}
+                {{ 'general.share.share' | t }}
+              </button>
+              <details>
+                <summary class="share-button__button button button--tertiary">
+                  {% render 'icon-share' %}
+                  {{ 'general.share.share' | t }}
+                </summary>
+                <div class="share-button__fallback">
+                  <div class="field">
+                    <input type="text"
+                          class="field__input"
+                          id="url"
+                          value="{{ shop.url | append: product.url }}"
```

Comment:
Here we need to change the url to the article :) 
```suggestion
                          value="{{ shop.url | append: article.url }}"
```

Reply:
doh.